### ISSUE TITLE
Scripts blocked on further website visits

### ISSUE THREAD

**Issue by effetiweb** [nicholasruggeri/cookies-enabler#7]
Hello!

I tried this script, it works well.
I noticed an issue.
When I go back on the website (please note clicked on the banner or scrolled in the page in the previous website visit) all scripts are blocked and not visible, such as Facebook box, Adsense, etc..

**Comment by gsimone** [comment]
Hi, it's appearing to work as intended. If you never agree explicitly to the use of cookies, they will not show.
Was the cookies banner visible on your second visit?

**Comment by effetiweb** [comment]
First of all thank you gsimone for your answer.

I show again the situation I came across when I tested it.

1) In the first visit to the website I see the banner and all scripts are blocked (Analytics, Facebook box, Adsense, etc..)
2) I accept clicking on the button on the banner
3)Scripts are downloaded in the page, ads appears.
4)I navigate to another page in the website, I don't see the banner (it's correct) but scripts are blocked. They should be allowed at this stage.
5)I leave the website and I navigate to another website.
6) Later, could be some days later as well. I go back to the website (no banner, it's correct because I gave the consent in the previous visit), although scripts are blocked.

I hope now, with this further explanation the issue I came across is much clear.

I would like to understand if I didn't setup the code well, or there are still some issue in the code that force to work correctly.

Thank you very much again.

**Comment by gsimone** [comment]
I tested the latest version of the script and it appears to be working as intended. Can you share the code you are using? There could be something missing.

**Comment by effetiweb** [comment]
Hi!

I carried out a new test with the latest version and I moved your script to the footer area of the website.
Now, it works correctly.
Before I inserted this script in the header and I got the error described in the previous message.
Thank you guys, good work!!!

I see you have in place to add the feature about iframe.

I would ask you, if you are thinking to add also a feature where the user can decide to activate cookies per type.
So, I give you an example, user can decide to activate only cookies about analytics, no activate cookies about advertising, etc...
Then, a box that give to the user in any moment the possibility to decide to opt out to a type of cookies.

Thank you again guys!
